### PR TITLE
Respawner Machine (Attempt Deux)

### PR DESCRIPTION
**Pull request by fludd12**
I was a dumb and couldn't do anything good in the last PR, so here's a real PR, with real text and commits.

### CODE REVIEW

File: code/game/objects/structures/misc.dm [hunk 1]
```diff
@@ -57,3 +57,19 @@
...
+	icon = 'icons/obj/xenoarchaeology.dmi'
+	icon_state = "borgcharger1(old)"
+	anchored = 1
+	density = 1
+
+/obj/structure/respawner/attack_ghost(mob/dead/observer/user as mob)
+	var/response = alert(user, "Are you sure you want to spawn like this?\n(If you do this, you won't be able to be cloned!)","Respawn?","Yes","No")
+	if(response == "Yes")
+		user.forceMove(get_turf(src))
+		log_admin("[key_name(user)] was incarnated by a respawner machine.")
```
Comment (FlattestGuitar) [inline code/game/objects/structures/misc.dm]: Should be `key_name_admin(user)` so we can follow and stuff.

File: code/game/objects/structures/misc.dm [hunk 2]
```diff
@@ -57,3 +57,19 @@
...
+		message_admins("[key_name(user)] was incarnated by a respawner machine.")
```
Comment (FlattestGuitar) [inline code/game/objects/structures/misc.dm]: Should be `key_name_admin(user)` so we can follow and stuff.

**Comment by Crazylemon64** [comment]
Ah, this looks like a really handy tool for adminbus! :+1:

**Comment by Spacemanspark** [comment]
Yay

**Pull request merged**